lib/modules/spoilerTags.js

```diff
@@ -0,0 +1,28 @@
+modules['spoilerTags'] = {
+	moduleID: 'spoilerTags',
+	moduleName: 'Global Spoiler Tags',
+	category: 'UI',
+	options: { },
+	description: 'Hides spoiler-tagged comments globally across Reddit',
```

Comment:
Yes, and that will include updating the description to reflect what the module can actually do.